esrally/rally.py

```diff
@@ -11,14 +11,12 @@
 from esrally import version, actor, config, paths, racecontrol, reporter, metrics, track, chart_generator, exceptions, time as rtime
 from esrally import PROGRAM_NAME, DOC_LINK, BANNER, SKULL, check_python_version
 from esrally.mechanic import team, telemetry
-from esrally.utils import io, convert, process, console, net
+from esrally.utils import io, convert, process, console, net, config as config_helper
```

Comment:
Maybe use a different name for this module? Instead of `config` maybe `opts` (because it is actually about handling command line options)?

Reply:
Good suggestion. Will address.